src/shogun/features/DenseFeatures.cpp

```diff
@@ -352,6 +355,12 @@ template<class ST> void CDenseFeatures<ST>::copy_feature_matrix(SGMatrix<ST> src
 	initialize_cache();
 }
 
+template<class ST> Eigen::Map < Eigen::Matrix< ST, -1, -1, 0,-1,-1>, 0, Eigen::Stride<0,0>> CDenseFeatures<ST>::get_eigen_map()
```

Comment:
The more I think about this, the more confused I am.  I also realise that everything I said above may not be correct - don't EigenMaps just wrap around SG structures?  What would the advantage be of getting a reference to the map?  Would I have to free that reference once it's no longer needed?  I've only learned C and I'm still learning C++, so I appreciate the help.